aws/app/lambda/nagware/lib/templates.js

```diff
@@ -45,12 +45,16 @@ const parseQueryResponse = (records) => {
   let formName = "";
 
   const firstRecord = records[0];
+  let isPublished = false;
 
   if (!process.env.AWS_SAM_LOCAL) {
     const jsonConfig = JSON.parse(firstRecord[2].stringValue.trim(1, -1)) || undefined;
     formName = firstRecord[1].stringValue !== "" ? firstRecord[1].stringValue : `${jsonConfig.titleEn} - ${jsonConfig.titleFr}`;
+    // TODO: Not sure what field the isPublished value is in, but based on the SQL query, it should be in the 4th index
```

Comment:
@craigzour this is a guess for me right now, I could use your guidance since I'm unable to test it

Reply:
Yep I agree. You can delete the comment ;)